src/circuit/gadget/sinsemilla/commit_ivk.rs

```diff
@@ -474,28 +474,24 @@ impl CommitIvkConfig {
                 {
                     let offset = 0;
                     // Copy in `ak`
```

Comment:
Consider deleting the "Copy in ..." and "Witness ..." comments in this method, which don't say anything that isn't obvious from the code.